modules/serverless-functions-secrets-configmaps-manually.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-manually_{context}"]
+= Adding function access to secrets and config maps manually
+
+You can manually add configuration for accessing secrets and config maps to your function.
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For each secret you want to mount as a volume, add the following YAML to the `volumes` section:
++
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+volumes:
+- secret: mysecret
+  path: /workspace/secret
+----
+
+. For each config map you want to mount as a volume, add the following YAML to the `volumes` section:
++
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+volumes:
+- configMap: myconfigmap
+  path: /workspace/configmap
+----
+
+. For each value from a secret key-value pair that you want to assign to an environment variable, add the following YAML to the `envs` section:
```

Comment:
I'd add a description for the fileds (what is the correct syntax for `value` field. Similar way (or a better one) as it is mentioned in upstream docs: odules/serverless-functions-secrets-configmaps-manually.adoc

The same for the other parts